### PR TITLE
test(material/slider): improve slider demo

### PR DESCRIPTION
**Pull request by wagnermaciel**

### CODE REVIEW

**Review by mmalerba — APPROVED** [review]
LGTM - probably shouldn't be "feat()" though

File: src/dev-app/slider/slider-demo.html [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+<mat-checkbox [color]="colorModel" [(ngModel)]="discrete">Discrete</mat-checkbox>
+<br>
+<mat-checkbox [color]="colorModel" [(ngModel)]="showTickMarks">Show tick marks</mat-checkbox>
+<br>
 <mat-button-toggle-group [(ngModel)]="colorModel" aria-label="Slider color options">
   <mat-button-toggle value="primary">Primary</mat-button-toggle>
   <mat-button-toggle value="accent">Accent</mat-button-toggle>
   <mat-button-toggle value="warn">Warn</mat-button-toggle>
 </mat-button-toggle-group>
+<br>
```
Comment (crisbeto) [inline src/dev-app/slider/slider-demo.html:10]: You didn't introduce this, but we might as well fix it while we're here. We shouldn't be using `br` tags for spacing.

File: src/dev-app/slider/slider-demo.ts [hunk 2]
```diff
@@ -26,6 +28,8 @@ import {MatButtonToggleModule} from '@angular/material/button-toggle';
   styleUrls: ['slider-demo.css'],
 })
 export class SliderDemo {
+  discrete: boolean = true;
```
Comment (crisbeto) [inline src/dev-app/slider/slider-demo.ts:31]: The `:boolean` here is redundant.

**Comment by wagnermaciel** [comment]
@mmalerba Would `docs()` be more appropriate? Also, here's a quick visual for what the demo looks like now
Screenshot 2022-11-17 at 9 37 45 AM

**Comment by mmalerba** [comment]
I can't remember if `docs` shows up in the changelog or not (we probably don't want this to show up since its not public facing docs), `test` would be a good option too, since its related to manual testing

**Pull request merged**